### PR TITLE
bug fix and reduce file size

### PR DESCRIPTION
**Pull request by wxupjack**
bug fix:
1. adapted to new leetcode.cn: leetcode has different sets of CSS for user login/off. The hard match('._1l1MA') and CSS selector is not available any more. After comparison, I select robust features in the page. `'div[data-track-load]')` and `$("a[href^='/problems/'][href$='/']")[1]`
2. fixed element missing when onload: wrap the `setTimeout` with a function outside, and extent waiting time because new leetcode takes longer to load.

feat:
- replace the scripts icon with leetcode's own
- set button style alone

---
bug修复：
1. 新版leetcode把之前的硬编码去掉了，有些地方用了随机id。同时，登陆前后的页面CSS也大有不同，所以只能通过`'div[data-track-load]')`和`$("a[href^='/problems/'][href$='/']")[1]`，这两个实验下来比较鲁棒的特征，来重新定位。
2. 修复了加载按钮时节点为undefined的bug，在外面又套了一层function。而且新版加载耗时更长，我电脑性能也不好。

feat：
- 图标直接用leetcode自己的favicon，减小点脚本体积。
- 把css独立出来了，方便后续调整。

### CODE REVIEW

File: leetcode-helper.js [hunk 1]
```diff
@@ -29,33 +29,51 @@
...
-        x.parentNode.appendChild(copyBtn);
-        x.parentNode.appendChild(generateSolutionBtn);
-    }, 1500);
-
+    window.onload = function () {
+        setTimeout(function () {
+            // 筛选出符合格式的href: '/problems/.*/'，使用两个selector串联，获取到的第一个为 /problems/，跳过
+            var x = $("a[href^='/problems/'][href$='/']")[1]
+            
+            // 等价于以下代码
```
Comment (tonngw) [inline leetcode-helper.js:61]: The commented code can be removed.

Comment (tonngw) [inline leetcode-helper.js:61]: The commented code can be removed.

File: leetcode-helper.js [hunk 2]
```diff
@@ -95,7 +113,9 @@
 
     function copyImpl() {
         // 题目描述 内容 Dom
-        var contentDom = $('._1l1MA')[0].outerHTML;
+        // var contentDom = $('div[data-track-load="description_content"]')[0].outerHTML; # 加上详细内容也可以
```
Comment (tonngw) [inline leetcode-helper.js:116]: This line code also can be removed.

**Review by tonngw — COMMENTED** [review]
Thank your pr very much, it is able to remove for the commented code. Formating the code before the next commit. I will merge your pr after you fix it. Thank you again for your submission.

File: leetcode-helper.js (hunk 1 above)
Comment (wxupjack) [reply leetcode-helper.js:61]: > with a function outside | 在外面又套了一层function

I found this is just unnecessary (my wrong understanding). Only need to extend waiting time for my poor computer. Ignore please.

Btw, can it loop to check whether `parentNode` exists, and append button until loaded.

Comment (tonngw) [reply leetcode-helper.js:61]: I also reply you on tampermonkey.

**Pull request closed without merging**